spec/support/shared_examples/resource_update.rb

```diff
@@ -7,12 +7,27 @@
     expect(updated_object._id).to eq resource_id
   end
 
-  it 'matches the informations sent' do
+  it 'matches the information sent' do
     updated_object = described_class.update(id: resource_id, params: params)
     params.keys.each { |key| expect(updated_object.send(key.to_s.to_snake_case)).to eq params[key] }
   end
 end
 
+shared_examples 'a singleton updatable resource returning a collection' do
+  around { |test| VCR.use_cassette("#{described_class::RESOURCE_NAME}_update") { test.run } }
+
+  it "returns a collection" do
+    updated_object = described_class.update(id: resource_id, params: params)
+    expect(updated_object).to be_a Array
```

Comment:
Nitpick: `expect(updated_object).to be_an Array` is the syntax for this we've used in other places (note the `be_an` instead of `be_a`

Reply:
🏅